dm/worker/config.go

```diff
@@ -55,6 +56,10 @@ func NewConfig() *Config {
 	fs.Int64Var(&cfg.Purge.Interval, "purge-interval", 60*60, "interval (seconds) try to check whether needing to purge relay log files")
 	fs.Int64Var(&cfg.Purge.Expires, "purge-expires", 0, "try to purge relay log files if their modified time is older than this (hours)")
 	fs.Int64Var(&cfg.Purge.RemainSpace, "purge-remain-space", 15, "try to purge relay log files if remain space is less than this (GB)")
+	fs.BoolVar(&cfg.Tracer.Enable, "tracer-enable", false, "whether to enable tracing")
+	fs.StringVar(&cfg.Tracer.TracerAddr, "tracer-tracer-addr", "", "tracing service rpc address")
```

Comment:
is `tracer-addr` enough? or something like `tracer-server-addr`

Reply:
This is only used in command line options, use `tracer-server-addr`